tests/unit/amo/components/TestAddonsByAuthorsCard.js

```diff
@@ -445,4 +446,26 @@ describe(__filename, () => {
     expect(root.find(AddonsCard))
       .toHaveProp('header', 'More add-ons by these developers');
   });
+
+  it('shows header text without More text', () => {
+    const root = renderAddonsWithType({
+      addonType: ADDON_TYPE_THEME,
+      showMore: false,
+      multipleAuthors: false,
+    });
+
+    expect(root.find(AddonsCard))
+      .toHaveProp('header', `Themes by ${fakeAuthor.name}`);
+  });
```

Comment:
I would like to see more test cases, ideally one for each addon type and number of authors.